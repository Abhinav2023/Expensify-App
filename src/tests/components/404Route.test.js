import React from 'react';
import {shallow} from  'enzyme';
import NotFoundPage from '../../components/404Router';

test("should render 404Not Found Page",()=>{
    const wrapper =shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})