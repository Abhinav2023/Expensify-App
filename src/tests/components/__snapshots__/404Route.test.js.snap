// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404Not Found Page 1`] = `
<div>
  This is from 404 component-
  <Link
    replace={false}
    to="/"
  >
    Go Home!
  </Link>
</div>
`;
